setup.py

```diff
@@ -5,7 +5,7 @@
 def main():
     setup(
         name='yelp_uri',
-        version='1.2.0',
+        version='2.0.0',
```

Comment:
that's what I'd do.  I generally avoid bumping versions in PRs anyway and leave it up to the maintainers to change the number and release (even when I'm the maintainer).  It also has the added benefit of avoiding merge conflicts with others :)